doc/HTTP-codes.txt

```diff
@@ -1,50 +0,0 @@
-Caching  Code
-	Successful 2xx
-c        200 OK
-         201 Created
-         202 Accepted
-c        203 Non-Authoritative Information *
-E        204 No Content
-         205 Reset Content *
-         206 Partial Content *
-	Redirection 3xx
-C        300 Multiple Choices
-C        301 Moved Permanently
-t        302 Moved Temporarily
--        303 See Other *
--        304 Not Modified
-E        305 Use Proxy (proxy redirect) *
-        Client Error 4xx
-E        400 Bad Request
--        401 Unauthorized
-         402 Payment Required *
-E        403 Forbidden
-E        404 Not Found
-E        405 Method Not Allowed *
-         406 Not Acceptable *
--        407 Proxy Authentication Required *
-         408 Request Timeout *
-         409 Conflict *
-C        410 Gone *
-         411 Length Required *
-         412 Precondition Failed *
-         413 Request Entity To Large *
-E        414 Request-URI Too Long *
-         415 Unsupported Media Type
-        Server Error 5xx
-E        500 Internal Server Error
-E        501 Not Implemented
-E        502 Bad Gateway
-E        503 Service Unavailable
-E        504 Gateway Timeout *
-         505 HTTP Version Not Supported *
-
-Notes:
-* HTTP 1.1
-c Cached unless a query response without expiry information
-C Cached
-E Negatively cached if no expiry headers
-t Cached only if expiry information
-- Not cached
-
-Unless other said, the response code is not cached.
```

Comment:
To (better) cover the removal of this and PROXY protocol files, I suggest simplifying PR description to say something like "Remove copies of publicly available documents and their derivatives".

Reply:
Added a reference to the IANA registry in the description for this.